### ISSUE TITLE
We need to add a smoothing algorithm to realtime shake contours

### ISSUE THREAD

**Issue by timlinux** [inasafe/inasafe-realtime#3]
We need to smooth MMI contours. @charlottemorgan could you start by giving us a bullet list of 'rules' about how the output should be smoothed? Can be in plain english perhaps with some before and after views of the same data. @akbargumbira can give you some of the generated datasets to play with if that helps.

See original ticket at https://github.com/inasafe/inasafe/issues/1201 for extended discussion.

**Comment by Charlotte-Morgan** [comment]
see #3

**Comment by akbargumbira** [comment]
@timlinux: Sorry, but I have no access to data anymore. Looking at the
problem, it seems that it could be solved with contour approximation. One
could implement it with this algorithm
https://en.m.wikipedia.org/wiki/Ramer–Douglas–Peucker_algorithm

**Comment by timlinux** [comment]
Hey @akbargumbira nice to see you here :-) This is just a migration of an old ticket.....I don't thing DP alg is what we are after here but more likely using a high/low pass filter on the raster before generating the contours.
